### ISSUE TITLE
Reindex a bib Record - Implement Alma functionality for bibliographic#update

### ISSUE THREAD

**Issue by christinach** [pulibrary/bibdata#748]
POST /bibliographic(.:format) bibliographic#update

This endpoint supports one-off, immediate indexing of a single marc record. This will be achieved by retrieving the record via the Alma API. Because the publishing jobs enrich the marc records before exporting them and the API endpoint does not, there are several places (enumerated below in child issues) where the record we got from the API will not have the data traject expects to find. To fix this, the bibliographic#update action should do additional API calls to retrieve that data, enriching the marc record so that it looks the same as a marc record we would get from the publishing job.

We've done something very similar in the `barcode#scsb` controller; that controller is a good starting point for determining what this implementation should look like. 

child issues:
- [x] catalog date #925 
- [ ] locations #1040
- [ ] electronic access #1039 
- [ ] call_number_browse_s and call_number_display https://github.com/pulibrary/marc_liberation/issues/967

 For each child ticket we'll need to determine how traject expects to find the data / how the publishing job enriches the record to add th data in question. The issues that were part of the [index from publishing jobs](https://github.com/pulibrary/marc_liberation/issues/806) epic may be helpful for that.

**Comment by hackartisan** [comment]
This is blocked on the traject configuration. Probably shouldn't start working on this until full indexing is in place.

**Comment by hectorcorrea** [comment]
Below are a few screenshots showing that fields adds to enrich the records during the Publishing job. To get the current fields login to Alma and go to Resources + Publishing Profiles + select "General Publishing via FTP" + "Edit".

![alma_enrich_1](https://user-images.githubusercontent.com/568286/106510274-b50aed00-649c-11eb-9471-3d2e095dff08.png)

![alma_enrich_2](https://user-images.githubusercontent.com/568286/106510286-b89e7400-649c-11eb-81e2-364495119f5d.png)

![alma_enrich_3](https://user-images.githubusercontent.com/568286/106510292-bb996480-649c-11eb-8054-596141e8a383.png)

![alma_enrich_4](https://user-images.githubusercontent.com/568286/106510305-bf2ceb80-649c-11eb-8e08-9aa3758e1406.png)

**Comment by hackartisan** [comment]
Unblocking this issue and blocking selected child issues instead.

**Comment by hackartisan** [comment]
closing in favor of #1478
